Makefile

```diff
@@ -147,7 +147,7 @@ test/profiler: $(GO_SRC) $(LIBBPF_HEADERS) $(LIBBPF_OBJ) bpf
 .PHONY: test
 ifndef DOCKER
 test: $(GO_SRC) $(LIBBPF_HEADERS) $(LIBBPF_OBJ) bpf test/profiler
-	$(go_env) go test -v $(shell go list ./... | grep -v "internal/pprof" | grep -v "pkg/profiler")
+	$(go_env) go test -run '[^TestIntegrationGRPC]' -v $(shell go list ./... | grep -v "internal/pprof" | grep -v "pkg/profiler")
```

Comment:
Is there a simpler way to do this? Ideally tags would be good but that would need changing all existing test files just to add the tag so used this hack instead.

Reply:
I think you can achieve this also just by tagging the integration test files. When running the unit tests, you can just exclude the tests that are tagged by integration.

Reply:
I would exclude it in the same way as we exclude the `internal/pprof` and `pkg/profiler` packages, that's how I see it done in most places.